frontend/src/App.scss

```diff
@@ -5,9 +5,17 @@ body {
 .links,
 .links:hover {
   text-decoration: none;
+}
+
+.links {
   color: black;
 }
 
+.btn-outline-dark:hover > .links,
+.faq-button:hover > .links {
```

Comment:
Looks good! the spinner is working as expected. One small comment is I don't think the faq-button class exists; the faq button has the same btn-outline-dark class as the Reviews button, and when I ran the app without this line, the styles were still applied so you can remove this line.